spec/spec

```diff
@@ -4,16 +4,6 @@
 
 #include "include/versions.h"
 
-beginseg
-    name "makerom"
-    // We set the address of the makerom segment as 0x80000400 - 0x1000, since the ROM header and IPL3 together
-    // are 0x1000 bytes long and we want the entry code to end up at address 0x80000400.
-    address 0x7FFFF400
-    include "$(BUILD_DIR)/src/makerom/rom_header.o"
-    include "$(BUILD_DIR)/src/makerom/ipl3.o"
-    include "$(BUILD_DIR)/src/makerom/entry.o"
-endseg
```

Comment:
`makerom` is done automagically now? How does it work?
I noted the Makefile has some special cases for the makerom files too, are those really needed too?